tests/Data/rsa-jwkset.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Use lowercase folder name `data`. Conventionally we use CamelCase names only when the folder contains class files.